src/xcoobee/sdk/Bees.js

```diff
@@ -1,247 +1,254 @@
-const ApiUtils = require('../../xcoobee/api/ApiUtils');

```

Comment:
I looks like all the line endings changed for this file. Let's revert the line endings back.

Please update your editor to preserve a file's existing line endings if possible.